schema/information/information_schema_2-0-0.json

```diff
@@ -26,7 +26,7 @@
       "description": "json message format version",
       "const": "1.99.99"
     },
-    "instance_id": {
+    "source_uuid": {
       "description": "unique id all other the world for a server",
```

Comment:
Could you also fix this error (note: it exists in many schemas): s/all other/all over/, please?

Reply:
Ok, fixed

Reply:
I don't think that Yann meant to replace `instance_id` by `source_uuid` as they do not represent the same information. He was referring to the spelling mistakes in the description.

Reply:
> I don't think that Yann meant to replace `instance_id` by `source_uuid` as they do not represent the same information. He was referring to the spelling mistakes in the description.

1. Indeed, I was speaking about the typo, and Hugues did fix it in the new iteration.
2. We still want to harmonise the top-level fields so that all messages still contain the same "header":
    * `message_type`: the type of message this is
    * `version`: the version of the schema that applies
    * `timestamp`: the UNIX timestamp the message was created at
    * `source_uuid`: the universally unique ID of the source that emitted the message
    * `origin`: the (type of) entity that emitted the message
    * `message`: the JSON-translation of the ETSI message

So far, only the schema for _information_ message defines an `instance_id`. If that is not the same thing as `source_uuid`, then we indeed must **add** `source_uuid` and keep `instance_id`.

Note that CAM, for example, has a `message.station_id`. Is that the same thing as the `instance_id` from the information message, or is it yet something else?